README.md

```diff
@@ -48,6 +48,8 @@ ADDITIONAL_OPS_FILES=""
 
 # use "test-results-go-cc" for CF with go-cf-api
 TEST_RESULTS_FOLDER=test-results
+# use "service_plans" for CF with go-cf-api
```

Comment:
No, the "TEST_SUITE_FOLDER" parameter is for selecting a subset of the tests. We used it for backfilling missing tests. Leave it empty for executing all tests. The "TEST_RESULTS_FOLDER" param is for separating the regular cloud controller results and the go results.

Reply:
Okay, I've changed the comment.